tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/util/BCompileUtil.java

```diff
@@ -72,6 +76,7 @@
 public class BCompileUtil {
 
     private static Path resourceDir = Paths.get("src/test/resources").toAbsolutePath();
+    private static final JBallerinaInMemoryClassLoader classLoader = new JBallerinaInMemoryClassLoader();
```

Comment:
shall we move this classLoader instance within the compileOnJBallerina method it self? Also shall we make the byte[] as the constructor argument for JBallerinaInMemoryClassLoader?